schematic_api/api/routes.py

```diff
@@ -608,8 +620,48 @@ def get_viz_tangled_tree_layers(schema_url, figure_type):
 
     return layers[0]
 
+def download_manifest(input_token, manifest_id, new_manifest_name='', as_json=None):
+    """
+    Download a manifest based on a given manifest id. 
+    Args:
+        input_token: token of asset store
+        manifest_syn_id: syn id of a manifest
+        newManifestName: new name of a manifest that gets downloaded.
+        as_json: boolean; If true, return a manifest as a json. Default to True
+    Return: 
+        file path of the downloaded manifest
+    """
+    # call config_handler()
+    config_handler()
+
+    # use Synapse Storage
+    store = SynapseStorage(input_token=input_token)
+
+    # default as_json to True
+    if as_json is None:
```

Comment:
I just realized this, is there a reason why you're not seeing `as_json=True` in the parameter section?

```
def download_manifest(input_token, manifest_id, new_manifest_name='', as_json=True)
```

Reply:
That's a good catch. Thanks Tom. For some reasons, I was under the impression that if users select nothing in swagger UI, as_json will be default back to None. But I just double checked, and it seems like if I set as_json = True in parameter, this is sufficient to default as_json = True.